topology/doc/user_manual_scripts/connections.py

```diff
@@ -24,6 +24,7 @@
 import nest
 import nest.topology as tp
 import matplotlib.pyplot as plt
+from mpl_toolkits.mplot3d.axes3d import Axes3D
```

Comment:
Is this used somewhere in the file? If it is, why was this `import` not here before?